device-types/LYNX Technik AG/RFR-5012.yml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: LYNX Technik AG
+model: RFR 5012
+slug: lynx-technik-ag-rfr-5012
+u_height: 2.0
+is_full_depth: true
+airflow: passive
```

Comment:
According to the datasheet the 5012 has integrated cooling with fans.

Reply:
hopefully correct now?